modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/dataframe.py

```diff
@@ -2566,3 +2596,8 @@ def _is_trivial_index(cls, index):
             and index.min() == 0
             and index.max() == len(index) - 1
         )
+
+
+def _check_join_supported(type):  # noqa: GL08
```

Comment:
I would suggest to move this function into `modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/utils.py`.

Reply:
Done.